### PR TITLE
black

### PR DESCRIPTION
**Pull request by itsvinayak**
### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: matrix/searching_in_sorted_matrix.py [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 
 def search_in_a_sorted_matrix(
-        mat: List[list], m: int, n: int, key: int or float) -> None:
-    '''
+    mat: List[list], m: int, n: int, key: int or float
```
Comment (cclauss) [inline matrix/searching_in_sorted_matrix.py:5]: This `int or float` thing does not really work, does it?!?

**Comment by cclauss** [comment]
If you as an authorized committer sync with master and then open a new PR then our autoblack GitHub Action should automatically blacken all code in the repo and build such commits.

File: matrix/matrix_operation.py [hunk 2]
```diff
@@ -14,8 +14,7 @@ def add(matrix_a: List[list], matrix_b: List[list]) -> List[list]:
     """
     if _check_not_integer(matrix_a) and _check_not_integer(matrix_b):
         _verify_matrix_sizes(matrix_a, matrix_b)
-        matrix_c = [[i + j for i, j in zip(m, n)]
-                    for m, n in zip(matrix_a, matrix_b)]
+        matrix_c = [[i + j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
```
Comment (cclauss) [inline matrix/matrix_operation.py:17]:
```suggestion
        return [[i + j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
```

File: matrix/matrix_operation.py [hunk 3]
```diff
@@ -171,12 +168,9 @@ def _verify_matrix_sizes(
...
     matrix_a = [[12, 10], [3, 9]]
     matrix_b = [[3, 4], [7, 4]]
-    matrix_c = [[11, 12, 13, 14], [21, 22, 23, 24],
-                [31, 32, 33, 34], [41, 42, 43, 44]]
+    matrix_c = [[11, 12, 13, 14], [21, 22, 23, 24], [31, 32, 33, 34], [41, 42, 43, 44]]
     matrix_d = [[3, 0, 2], [2, 0, -2], [0, 1, 1]]
-    print(
-        f"Add Operation, {matrix_a} + {matrix_b}"
-        f" = {add(matrix_a, matrix_b)} \n")
+    print(f"Add Operation, {matrix_a} + {matrix_b}" f" = {add(matrix_a, matrix_b)} \n")
```
Comment (cclauss) [inline matrix/matrix_operation.py:173]:
```suggestion
    print(f"Add Operation, {matrix_a} + {matrix_b} = {add(matrix_a, matrix_b)} \n")
```

File: matrix/matrix_operation.py [hunk 4]
```diff
@@ -28,8 +27,7 @@ def subtract(matrix_a: List[list], matrix_b: List[list]) -> List[list]:
     """
     if _check_not_integer(matrix_a) and _check_not_integer(matrix_b):
         _verify_matrix_sizes(matrix_a, matrix_b)
-        matrix_c = [[i - j for i, j in zip(m, n)]
-                    for m, n in zip(matrix_a, matrix_b)]
+        matrix_c = [[i - j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
```
Comment (cclauss) [inline matrix/matrix_operation.py:30]:
```suggestion
        return [[i - j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
```

File: matrix/matrix_operation.py [hunk 5]
```diff
@@ -101,8 +99,8 @@ def minor(matrix: List[list], row: int, column: int) -> List[list]:
     >>> minor([[1, 2], [3, 4]], 1, 1)
     [[1]]
     """
-    minor = matrix[:row] + matrix[row + 1:]
-    minor = [row[:column] + row[column + 1:] for row in minor]
+    minor = matrix[:row] + matrix[row + 1 :]
+    minor = [row[:column] + row[column + 1 :] for row in minor]
```
Comment (cclauss) [inline matrix/matrix_operation.py:103]:
```suggestion
    return [row[:column] + row[column + 1 :] for row in minor]
```

File: matrix/searching_in_sorted_matrix.py (hunk 1 above)
Comment (itsvinayak) [reply matrix/searching_in_sorted_matrix.py:5]: it does not produce any error, but should I change to `float ` from `int or float`

File: matrix/matrix_operation.py [hunk 6]
```diff
@@ -14,8 +14,7 @@ def add(matrix_a: List[list], matrix_b: List[list]) -> List[list]:
     """
     if _check_not_integer(matrix_a) and _check_not_integer(matrix_b):
         _verify_matrix_sizes(matrix_a, matrix_b)
-        matrix_c = [[i + j for i, j in zip(m, n)]
-                    for m, n in zip(matrix_a, matrix_b)]
+        return [[i + j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
         return matrix_c
```
Comment (cclauss) [inline matrix/matrix_operation.py:18]:
```suggestion
```

File: matrix/matrix_operation.py [hunk 7]
```diff
@@ -28,8 +27,7 @@ def subtract(matrix_a: List[list], matrix_b: List[list]) -> List[list]:
     """
     if _check_not_integer(matrix_a) and _check_not_integer(matrix_b):
         _verify_matrix_sizes(matrix_a, matrix_b)
-        matrix_c = [[i - j for i, j in zip(m, n)]
-                    for m, n in zip(matrix_a, matrix_b)]
+        return [[i - j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
         return matrix_c
```
Comment (cclauss) [inline matrix/matrix_operation.py:31]:
```suggestion
```

File: matrix/matrix_operation.py [hunk 8]
```diff
@@ -101,8 +99,8 @@ def minor(matrix: List[list], row: int, column: int) -> List[list]:
     >>> minor([[1, 2], [3, 4]], 1, 1)
     [[1]]
     """
-    minor = matrix[:row] + matrix[row + 1:]
-    minor = [row[:column] + row[column + 1:] for row in minor]
+    minor = matrix[:row] + matrix[row + 1 :]
+    return [row[:column] + row[column + 1 :] for row in minor]
     return minor
```
Comment (cclauss) [inline matrix/matrix_operation.py:104]:
```suggestion
```

**Comment by itsvinayak** [comment]
> If you as an authorized committer, sync with master and open a new PR, then our autoblack GitHub Action should automatically blacken all code in the repo and then generate a similar commit.

@cclauss this branch in even with master branch

File: matrix/searching_in_sorted_matrix.py (hunk 1 above)
Comment (cclauss) [reply matrix/searching_in_sorted_matrix.py:5]: My sense is that it should be something like `Union[int, float]` and in the import section add, `from typing import Union` but please read https://docs.python.org/3/library/typing.html#typing.Union to be sure.

File: matrix/matrix_operation.py (hunk 8 above)
Comment (cclauss) [reply matrix/matrix_operation.py:102]: This line is _unreachable code_

File: matrix/searching_in_sorted_matrix.py [hunk 9]
```diff
@@ -1,8 +1,8 @@
-from typing import List
+from typing import List, Union
 
 
 def search_in_a_sorted_matrix(
-    mat: List[list], m: int, n: int, key: int or float
+    mat: List[list], m: int, n: int, key: Union[int, float]
```
Comment (itsvinayak) [inline matrix/searching_in_sorted_matrix.py:5]: @cclauss have a look at it

**Pull request merged**